go/common/node/node.go

```diff
@@ -109,6 +112,15 @@ const (
 	// RoleReserved are all the bits of the Oasis node roles bitmask
 	// that are reserved and must not be used.
 	RoleReserved RolesMask = ((1 << 32) - 1) & ^((RoleStorageRPC << 1) - 1)
+
+	RoleComputeWorkerName = "compute"
+	RoleStorageWorkerName = "storage"
+	RoleKeyManagerName    = "key-manager"
+	RoleValidatorName     = "validator"
+	RoleConsensusRPCName  = "consensus-rpc"
+	RoleStorageRPCName    = "storage-rpc"
+
+	RolesMaskStringSep = ","
```

Comment:
This should probably be package private (lower case)

Reply:
I don't have strong preferences about this. Just noting that, for example, staking threshold names are also public:
https://github.com/oasisprotocol/oasis-core/blob/d1b69f5aac431eac05705cae0297936ad410a3eb/go/staking/api/api.go#L651-L657

Reply:
Yeah, I ment this only for the separator. I think others should be public.

Reply:
Oh, I didn't understand the comment was only referring to the separator :slightly_smiling_face:.
Yeah, I'll make the separator private. Thanks for catching this!

Reply:
Done.